locales/sv_SE/pcsx2_Main.po

```diff
@@ -1320,7 +1316,9 @@ msgstr "SystemInställningsDialog - %s"
 
 #: pcsx2/gui/Dialogs/SysConfigDialog.cpp:246
 msgid "Plugins"
-msgstr "InsticksProgram"
+msgstr ""
+"Insticks\n"
+" Program"
```

Comment:
What's with the newline? Also while insticksprogram is recommended by TNC, I'm not sure it's the right choice for translations when 'plugin' is a legitimate Swedish word.

Reply:
The newline makes it better presented in the dialog.
TNC?
I for one prefere not to use "plugin" over "insticksprogram".